forui/lib/src/widgets/breadcrumb.dart

```diff
@@ -0,0 +1,284 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+import 'package:forui/forui.dart';
+import 'package:meta/meta.dart';
+
+/// A Breadcrumb.
+///
+/// A breadcrumb is a list of links that help visualize a page's location within a site's hierarchical structure,
+/// it allows navigation up to any of the ancestors.
+///
+/// See:
+/// * https://forui.dev/docs/navigation/breadcrumb for working examples.
+/// * [FBreadcrumbStyle] for customizing a breadcrumb's appearance.
+final class FBreadcrumb extends StatelessWidget {
+  /// The breadcrumb's style. Defaults to the appropriate style in [FThemeData.breadcrumbStyle].
+  final FBreadcrumbStyle? style;
+
+  /// The list of breadcrumb items.
+  ///
+  /// Each item in the list is typically an [FBreadcrumbItem], with a [divider] placed between consecutive items.
+  /// The last item in the list is usually an [FBreadcrumbItem] with the `selected` property set to true,
+  /// to indicate the current page in the breadcrumb.
```

Comment:
`/// to indicate the current page in the breadcrumb.`

The phrasing for this portion is off.